.github/workflows/build.yml

```diff
@@ -5,38 +5,38 @@ on: push
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
My understanding was that we still wanted to deploy every commit to `latest` and just pin the tags

Reply:
Sorry, I have no idea what you mean. We push the `latest` tag to be the tag that's created, as discussed?

Reply:
From what @david-cooke said, I thought the plan was that every commit was deployed to the `latest` tag, and only when we tag a new version do we create a new Docker tag. This would only do the latter.

Reply:
Yes, there seemed to have been a misunderstanding there. This will push `latest` on every tag push.